logging/slog_logger.go

```diff
@@ -1,75 +1,119 @@
 package logging
 
 import (
+	"context"
 	"fmt"
+	"io"
 	"log/slog"
 	"os"
+	"runtime"
+	"time"
+)
+
+type SlogHandlerType string
+
+const (
+	SLogHandlerTypeJson     SlogHandlerType = "json" // prints debug and above
+	SLogHandlerTypeJsonText SlogHandlerType = "text" // prints info and above
```

Comment:
Are they used?

Reply:
Nope, good catch. Originally wanted this as an option to the constructor, but opted for 2 separate constructors. Removed in [c276964](https://github.com/Layr-Labs/eigensdk-go/pull/127/commits/c2769642573052270294ccbdc108357f311b3118)